src/test/store/publish.test.js

```diff
@@ -233,10 +233,32 @@ describe('attemptToPublish', function() {
       expect(publishResult).toBe(true);
     }
 
+    async function waitUntilData<T>(
+      predicate: () => Promise<T> | T,
+      times: number = 10
+    ): Promise<T> {
+      function wait() {
+        return new Promise(resolve => setTimeout(resolve, 50));
```

Comment:
I used "50ms" following the example of testing-library's waiting utilities. (in a first version I used 0, which worked too, but maybe it takes too much CPU for nothing)

Reply:
Optional: This may be useful to put in `src/test/fixtures/utils.js` with a quick code comment, otherwise it's fine here as is.

Reply:
I moved it, and changed it that it doesn't accept null values either. (previously this was only undefined)